test-support/picolibc-test-wrapper.py

```diff
@@ -24,23 +24,7 @@
     "picolibc_armv8.1m.main_hard_nofp_mve-build/test/fenv",
     "picolibc_armv8.1m.main_hard_nofp_mve-build/test/math_errhandling",
 
-    "picolibc_armv8m.main_hard_fp-build/test/printf_scanf",
-    "picolibc_armv8m.main_hard_fp-build/test/printff_scanff",
-    "picolibc_armv8m.main_hard_fp-build/test/printff-tests",
-    "picolibc_armv8m.main_hard_fp-build/test/math_errhandling",
-    "picolibc_armv8m.main_hard_fp-build/test/rounding-mode",
-    "picolibc_armv8m.main_hard_fp-build/test/long_double",
-    "picolibc_armv8m.main_hard_fp-build/test/rand",
-    "picolibc_armv8m.main_hard_fp-build/test/fenv",
-    "picolibc_armv8m.main_hard_fp-build/test/math-funcs",
-    "picolibc_armv8m.main_hard_fp-build/test/test-strtod",
-    "picolibc_armv8m.main_hard_fp-build/test/test-efcvt",
-    "picolibc_armv8m.main_hard_fp-build/test/complex-funcs",
-    "picolibc_armv8m.main_hard_fp-build/test/semihost/semihost-times",
-    "picolibc_armv8m.main_hard_fp-build/newlib/libm/test/math_test",
-    "picolibc_armv8m.main_hard_fp-build/test/libc-testsuite/sscanf",
-    "picolibc_armv8m.main_hard_fp-build/test/libc-testsuite/strtod",
-    "picolibc_armv8m.main_hard_fp-build/test/test-fma"
+    "picolibc_armv8m.main_hard_fp-build/test/math_errhandling"
```

Comment:
If this remaining tests is failing due to missing floating point exception support in compiler-rt, you could put it under the commend above, together with other test failing for the same reason.